web/public/mgmt/config.yml

```diff
@@ -658,13 +658,85 @@ collections:
                             widget: object
                             fields:
                               - { label: "Description", name: "description", widget: "markdown" }
+                  - label: Trustees
+                    name: trustees
+                    collapsed: true
+                    widget: object
+                    fields:
+                      - { label: "Label", name: "label", widget: "string" }
+                      - { label: "Add Button Text", name: "addButtonText", widget: "string" }
+                      - { label: "Error", name: "error", widget: "string" }
+                      - { label: "Success snackbar - body", name: "successSnackbarBody", widget: "string" }
```

Comment:
[pebble] Not just here but in all your CMS changes - if something is longer than a few words, it should be a "text" widget instead of a string widget so that the content team can edit in a text area. 

(obviously the comment about Markdown takes precedence here though)

Overall, I recommend looking opening up the CMS to review your changes. If YOU can't edit the whole text in the small "string" box or if you can't edit the Contextual Info / bolding via rich text editor, the widget may need to be updated.